src/rgw/rgw_sal_motr.cc

```diff
@@ -2638,7 +2642,8 @@ int MotrObject::get_bucket_dir_ent(const DoutPrefixProvider *dpp, rgw_bucket_dir
 
 out:
   if (rc == 0) {
-    decode(attrs, iter);
+    sal::Attrs dummy;
```

Comment:
This change is again reverting Rahul's changes ? https://github.com/Seagate/cortx-rgw/pull/246/files

Reply:
this is reverted now.